src/actions/SensorDetailActions.js

```diff
@@ -0,0 +1,27 @@
+export const SENSOR_DETAIL_ACTIONS = {
+  UPDATE_NAME: 'SensorDetail/updateName',
+  UPDATE_CODE: 'SensorDetail/updateCode',
+  UPDATE_LOG_INTERVAL: 'SensorDetail/updateLogInterval',
+  UPDATE_CONFIG: 'SensorDetail/updateConfig',
+};
+
+const updateName = name => ({ type: SENSOR_DETAIL_ACTIONS.UPDATE_NAME, payload: { name } });
+
+const updateCode = code => ({ type: SENSOR_DETAIL_ACTIONS.UPDATE_CODE, payload: { code } });
+
+const updateLogInterval = logInterval => ({
+  type: SENSOR_DETAIL_ACTIONS.UPDATE_LOG_INTERVAL,
+  payload: { logInterval },
+});
+
+const updateConfig = (configType, configField, value) => ({
+  type: SENSOR_DETAIL_ACTIONS.UPDATE_CONFIG,
+  payload: { configType, configField, value },
+});
+
+export const SensorDetailActions = {
```

Comment:
- This is all very similar to the New Sensor actions/reducer.

Maybe the state shape can be separate. I was thinking you could use `combineReducers` on the two state shapes for `sensorDetail` and `newSensor` under a common `sensor` header.

Regardless, this would still create separate actions. I went with this because I think it just makes the features separate: the 'new sensor flow' is a bit different than a normal create/edit feature?